language/de-DE/tpl_cassiopeia.ini

```
@@ -4,14 +4,23 @@
 ; License GNU General Public License version 2 or later; see LICENSE.txt, see LICENSE.php
 ; Note : All ini files need to be saved as UTF-8
 
-TPL_CASSIOPEIA_XML_DESCRIPTION="Weiterentwicklung der Space-Themes (Solarflare in 1.0, Milkyway in 1.5 und Protostar in 3.0), Cassiopeia ist das Joomla 4 Frontend Template basierend auf Bootstrap 4."
 
 CASSIOPEIA="Cassiopeia Frontend Template"
 TPL_CASSIOPEIA_BACKTOTOP="Zurück nach oben"
 TPL_CASSIOPEIA_BACKTOTOP_LABEL="„Zurück nach oben“ Link"
+TPL_CASSIOPEIA_COLOR_NAME_ALTERNATIVE="Alternativ"
+TPL_CASSIOPEIA_COLOR_NAME_LABEL="Theme Farbe"
+TPL_CASSIOPEIA_COLOR_NAME_STANDARD="Standard"
 TPL_CASSIOPEIA_FLUID="Fluid"
 TPL_CASSIOPEIA_FLUID_LABEL="Layout"
+TPL_CASSIOPEIA_FONT_GROUP_LOCAL="Schriften aus Verzeichnis"
+TPL_CASSIOPEIA_FONT_GROUP_WEB="Schriften aus dem Web"
+TPL_CASSIOPEIA_FONT_LABEL="Schriften-Schema"
+TPL_CASSIOPEIA_FONT_NOTE_TEXT="Das Laden von Schriften aus externen Quellen kann in einigen Ländern gegen Datenschutzbestimmungen verstoßen.<br>Das Laden von Schriften aus einem lokalen Ordner kann sich auf die Leistung der Website auswirken."
```

```suggestion
TPL_CASSIOPEIA_FONT_NOTE_TEXT="Das Laden von Schriften aus externen Quellen kann in einigen Ländern gegen Datenschutzbestimmungen verstoßen."
```

Die aussage ist Blödsinn da es keinen Vorteil gibt wenn man von Google läd da die pro Seite immer neu geladen werden. Daer gibt es keinen Unterschied eher ist das lokale laden schneller :D